compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -266,6 +269,7 @@ private sealed trait WarningSettings:
          |Examples:
          |  - change every warning into an error: -Wconf:any:error
          |  - silence deprecations: -Wconf:cat=deprecation:s
+         |  - silence warnings in src_managed directory: -Wconf:src=src_managed/.*
```

Comment:
Shouldn't this example end with `:s` or `:silent`?

Reply:
You're correct, fixed https://github.com/lampepfl/dotty/pull/18783/files#diff-05e59408426c08d96c341addfb7f7a06353bd762027e856e8d4c1e8719e5c6bdR277